src/Core/src/Handlers/View/AbstractViewHandler.cs

```diff
@@ -13,7 +13,9 @@
 
 namespace Microsoft.Maui.Handlers
 {
-	public abstract partial class AbstractViewHandler<TVirtualView, TNativeView> : IViewHandler
+	public abstract partial class AbstractViewHandler<TVirtualView, TNativeView> : 
+		ViewHandler,
```

Comment:
Maybe? The idea here is that now I call up into disconnect/connect on ViewHandler for generic ViewHandler needs

Or it could not use inheritance and just call a static dissconnect/connect

There is something natural feeling though about AbstractViewHandler inheriting from ViewHandler